clang/include/clang/Basic/CodeGenOptions.def

```diff
@@ -480,6 +480,9 @@ CODEGENOPT(DisableSYCLEarlyOpts, 1, 0)
 /// which do not contain "user" code.
 CODEGENOPT(OptimizeSYCLFramework, 1, 0)
 
+/// Whether to use alloca address space for `sret` arguments.
+CODEGENOPT(UseAllocaASForSrets, 1, 0)
+
```

Comment:
Per the `TODO` message in `clang/lib/Driver/ToolChains/Clang.cpp` regarding a temporary fix, should there likewise be a comment here that this CG option is temporary? Or is this part of what will be a permanent fix?

Reply:
Fair enough, I will add it here too.